examples/bind/ecommerce.amp.html

```diff
@@ -81,14 +145,18 @@
       - Force quotation marks for literal keys, AND
       - Reintroduce variables in object keys (removed in #7582)
 
-    <amp-state [src]="..." on="fetch:AMP.setState({"shirts": {selectedIndex: event.response}})"
+    <amp-state [src]="..." on="fetch:AMP.setState({"shirts": {selected: {id: event.response}}})"
   -->
-  <amp-state id="shirts" [src]="'/bind/ecommerce/sizes?shirt=' + selectedIndex">
+  <amp-state id="shirts" [src]="'/bind/ecommerce/sizes?shirt=' + selected.id">
     <script type="application/json">
       {
         "0": {
           "color": "black",
-          "image": "./shirts/black.jpg"
+          "image": "./shirts/black.jpg",
+          "sizes": {
```

Comment:
Maybe be a little more explicit in the comments above about why '0' has sizes but the others don't?